modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectActionLocalServiceTest.java

```diff
@@ -199,6 +301,8 @@ public void testAddObjectAction() throws Exception {
 				objectActionLabelException.getMessage());
 		}
 
+		// Name is null
```

Comment:
Will Brian approve these comments? Since the we already have the message on the exception check?